corehq/messaging/smsbackends/airtel_tcl/models.py

```diff
@@ -9,7 +9,7 @@
 from corehq.util.timezones.conversions import ServerTime
 from datetime import datetime
 from django.conf import settings
-
+import re
```

Comment:
this would need to go on top of the file. We use [isort](https://pypi.org/project/isort/) to help us take those decisions for us
`isort <file_path>`